src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -610,17 +613,17 @@ private static string ConvertEndOfLineAndQuotationCharactersToEscapeForm(string
         other.SymbolName == SymbolName &&
         other.Ordinal == Ordinal &&
         other.IsOptional == IsOptional &&
-        SymbolEqualityComparer.Default.Equals(other.Type, Type);
+        SymbolEqualityComparer.IncludeNullability.Equals(other.Type, Type);
```

Comment:
Prior to the interceptors feature, we couldn't generate separate overloads for endpoints that had the same parameter type but differing nullability (see https://github.com/dotnet/aspnetcore/issues/46622). Interceptors don't have the same problem so we can generate code for endpoints that use the same type but differing nullability.